src/components/Review.jsx

```diff
@@ -1,19 +1,27 @@
 import { useParams } from "react-router-dom";
-import { fetchReviews } from "./api";
+import { fetchReviews, fetchReviewComments } from "./api";
 import { useEffect, useState } from "react";
 
 const Review = () => {
 	const { review_id } = useParams();
 	const [review, setReview] = useState({});
+	const [comments, setComments] = useState([]);
 	const [loading, setLoading] = useState(false);
```

Comment:
as well as loading, we should have a state for if there is an error to prevent the user waiting for what would be forever